### DISCUSSION TITLE
blog/js/check-for-at-least-one-capital-letter-in-js/

### DISCUSSION THREAD

**Discussion by giscus** [GitHubGanKai/blog#4] [Announcements]
# blog/js/check-for-at-least-one-capital-letter-in-js/

Learn efficient methods to verify if a string contains at least one capital letter in JavaScript using regular expressions, built-in methods, and custom functions.

https://geekskai.com/blog/js/check-for-at-least-one-capital-letter-in-js/

**Comment by gankaicode** [comment]
Very cool, very professional blog, worth learning from.
